server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultAcceptOptionsContext.java

```diff
@@ -66,14 +71,18 @@
 
     private final Map<String, String> binds;        // gets modified by balancer service
     private final Map<String, String> options;      // unmodifiable map without bind options like tcp.bind etc
+    private Properties configuration;
 
     public DefaultAcceptOptionsContext() {
         this.binds = new HashMap<>();
         this.options = Collections.emptyMap();
     }
 
-    public DefaultAcceptOptionsContext(ServiceAcceptOptionsType acceptOptions, ServiceAcceptOptionsType defaultOptions) {
+    public DefaultAcceptOptionsContext(ServiceAcceptOptionsType acceptOptions, ServiceAcceptOptionsType defaultOptions, Properties configuration) {
         Map<String, String> options = parseAcceptOptionsType(acceptOptions);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "options" which hides the field declared at line 73. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaazing.wan:9000/coding_rules#rule_key=squid%3AHiddenFieldCheck)